tool/resources/org/antlr/v5/tool/templates/codegen/Kotlin/Kotlin.stg

```diff
@@ -226,6 +229,11 @@ Parser(parser, funcs, atn, sempredFuncs, superClass) ::= <<
 Parser_(parser, funcs, atn, sempredFuncs, superClass) ::= <<
 @Suppress("UNUSED_VARIABLE", "ClassName", "FunctionName", "LocalVariableName", "ConstPropertyName", "ConvertSecondaryConstructorToPrimary")
 public open class <parser.name>(input: TokenStream) : <superClass; null="Parser">(input) {
+
+	init {
+		addErrorListener(ConsoleErrorListener())
```

Comment:
because originally the default error listener is the ConsoleErrorListener, which cannot be part of core because it requires IO. This ensures a backwards compatible behavior